projects/public-search/src/app/document-brief/document-brief.component.html

```diff
@@ -41,7 +41,7 @@ <h4 class="card-title mb-1">
             <span *ngIf="!contribution.agent.pid">
               {{ contribution | contributionFormat }}
             </span>
-            <a *ngIf="contribution.agent.pid" [routerLink]="['/records', 'persons', 'detail', contribution.agent.pid]">
```

Comment:
you still need to use `routerLink`

Reply:
I tried that first:
`<a *ngIf="contribution.agent.pid" [routerLink]="['/', viewcode, 'persons', contribution.agent.pid]" >`
but it raise an error : `Cannot match any routes`. I don't know how to fix it quickly in public-search.
So, I use `[href]`instead. It's the same method used for the document title link.

Reply:
You are right this route is not an angular route. Sorry.